src/pudl/output/denorm_eia.py

```diff
@@ -4,6 +4,78 @@
 
 import pudl
 from pudl.metadata.fields import apply_pudl_dtypes
+from pudl.output.eia860 import (
+    fill_generator_technology_description,
+    fill_in_missing_ba_codes,
+)
+
+logger = pudl.logging_helpers.get_logger(__name__)
+
+
+@asset(io_manager_key="pudl_sqlite_io_manager", compute_kind="Python")
+def pu_eia(
```

Comment:
Is `pu_eia` really just an intermediary table that's used to build up other denormalized tables? Or does it have stand-alone use cases for which we need to distribute it in the DB? If it's just a helper for building others, maybe it would be appropriate to persist it using the default (pickled dataframe on disk) IO Manager?